impress-org/givewp
# Feature: Add new 'Recurring' tab and 'Addon' button to settings page

## Description

This PR improves the Recurring add-on tab on the Settings page with a new design. It should only display if Recurring is NOT active. The tab itself has been repurposed into an external link. When selected the user is now directed to the Recurring sales page in favor of the previous functionality which registered and loaded details directly on the Settings page. 

In addition to the changes listed above if the user has the Recurring add-on active, a new 'Add-on' button is displayed on the far right of Tab list. It is also an external link that opens up the "Must have" addons page in a new window when clicked. These changes improve the user experience by providing clear guidance on how to install the Recurring add-on, and makes it easy to access the add-on page for those who already have it installed.

Please Reference repo: https://github.com/impress-org/give-recurring/pull/1181 for additional information and changes needed to fully test out the Settings Page.

## Affects
Donations -> Settings Page

## Visuals
https://www.loom.com/share/7a46b004354143b4b73a4def41974fe8

## Testing Instructions

- Without Recurring add-on active visit the settings page. It can be found by selecting Donations->Settings from the side navigation.
- Verify the new Recurring tab design displays and when clicked it opens up the Recurring add-on documentation page in a new window.

- With Recurring NOT active visit the settings page again.
- Verify the Recurring tab no longer displays and that a new "Add-on" button does display to the far-most right of the Tab list. When clicking this link it will open the add-on page in a new window.


## Pre-review Checklist
